CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed a crash in `tt aeon connect` when processing responses
   from certain SQL commands.
 - `tt cat|play <DIR>` with directories handles only `.snap` or `.xlog` files.
+- `pack` with modules include only internal modules in the `tt` environment.
```

Comment:
May be it's better to use 'under tt root directory' or something similar to avoid confusing, because _internal module_ has special meaning in tt universe.